libs/model/src/models/address.ts

```diff
@@ -101,6 +101,7 @@ export default (
       indexes: [
         { fields: ['address', 'community_id'], unique: true },
         { fields: ['user_id'] },
+        // TODO: maybe add index on mod/admin
```

Comment:
let's resolve this TODO by adding the index or removing the comment

Reply:
It really should be an XXX or note of some sort, we don't want to modify the schema unless there's performance issues querying mods/admins, but we also don't want to lose the idea that we can simply add an index... I will update it.